.holo/sources/slate.toml

```diff
@@ -1,3 +1,3 @@
 [holosource]
-url = "https://github.com/SlateFoundation/slate"
+url = "https://github.com/nbey/slate"
```

Comment:
@nbey as of [hologit v0.14.0](https://github.com/JarvusInnovations/hologit/releases/tag/v0.14.0) you can set `HOLO_SOURCE_SLATE="https://github.com/nbey/slate"` in your environment to override this